background_scripts/main.coffee

```diff
@@ -455,138 +452,58 @@ updatePositionsAndWindowsForAllTabsInWindow = (windowId) ->
         openTabInfo.positionIndex = tab.index
         openTabInfo.windowId = tab.windowId)
 
-splitKeyIntoFirstAndSecond = (key) ->
-  if (key.search(namedKeyRegex) == 0)
-    { first: RegExp.$1, second: RegExp.$2 }
-  else
-    { first: key[0], second: key.slice(1) }
-
-getActualKeyStrokeLength = (key) ->
-  if (key.search(namedKeyRegex) == 0)
-    1 + getActualKeyStrokeLength(RegExp.$2)
-  else
-    key.length
+splitByKeys = (key) ->
+  returnArray = []
+  while key
+    if (key.search(namedKeyRegex) == 0)
+      returnArray.push RegExp.$1
+      key = RegExp.$2
+    else
+      returnArray.push key[0]
+      key = key[1..]
+  returnArray
```

Comment:
We may rework `namedKeyRegex` into:
``` Regex
/<(?:(?:a-(?:c-)?(?:m-)?|c-(?:m-)?|m-)([A-Z][0-9A-Z]+|[a-z][0-9a-z]+|[^\s])|[A-Z][0-9A-Z]+|[a-z][0-9a-z]+)>|[^\s]/g
```

Then we can use `key.match(namedKeyRegex)` to get the splitted array easily.

PS: If we have to considering that `"<space>"` is changed into `" "`, then its end `|[^\s]/g` should be `|./g`.